datafusion/src/logical_plan/plan.rs

```diff
@@ -274,6 +283,7 @@ impl LogicalPlan {
             LogicalPlan::Extension { node } => node.schema(),
             LogicalPlan::Union { schema, .. } => schema,
             LogicalPlan::CreateMemoryTable { input, .. } => input.schema(),
+            LogicalPlan::DropTable { schema, .. } => schema,
```

Comment:
We could return an empty schema here instead of keeping it the plan node.

Reply:
I tried creating an empty schema before, but `schema` returns `&DFSchemaRef`, so rustc complains about a reference to dropped temporary object.

Reply:
yeah, that is unfortunate but I think the comments in the `LogicalPlan` struct are reasonable

Reply:
Hm. Another solution could be having / referencing a static object for an empty schema, but I'm ok with the current situation.